source/developers/how-tos/maintain-a-repo.rst

```diff
@@ -111,6 +111,28 @@ Update your readme until it meets the above specification.
 
 .. _readme specification: https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0055/decisions/0003-readme-specification.html
 
+Your CCs are Aligned With Your Expectations as Maintainer
+*********************************************************
+
+This is a more difficult item to test but a repo that you take on may already
+have other CCs that have merge rights on it.  You should review which CCs have
+write access to your repo on the `CCs Wiki Page`_.  You should communicate with
```

Comment:
FYI, I searched this page for `dashboard` and thought there were no CC's yet. 😅 

I'm not opposed to the `all_frontends` group, and it makes sense in particular for the translations / shared infrastructure, but would it be possible to have a more reliable way to look up CC's? e.g. can I see them somewhere in github?

Alternatively we could just have a more explicit handoff process. Like, when you transition ownership, make a discourse post (maybe with a template where you tag all the CCs) about it and have a conversation with people about work in flight.

Reply:
I added a not about it now, long term it would be nice to just be able to get a list given a repo.

Reply:
I think it could be a good idea to define what groups like `all_frontends` at the top of the wiki page so your ctl-f would hit on your repo. It may be difficult to keep in sync if, for example, we added a new MFE but forgot to update the wiki page. IDK. All imperfect solutions.

Reply:
we provide, at the top of the core contributors page, links to all the github groups that we create for core contributors. for example I see three already for frontend.

<img width="339" alt="image" src="https://github.com/openedx/docs.openedx.org/assets/1985317/38ab4f33-2565-48de-a246-1d0f8ecafd05">

Reply:
ALSO, Axim admins can just tell you which groups have write access to your repo. Very possibly the easiest/best course of action is to just file an Axim request ticket. For example if I go into settings on `frontend-app-profile` I see this list (which is a part of the whole)

<img width="803" alt="image" src="https://github.com/openedx/docs.openedx.org/assets/1985317/1481836b-b605-44a0-86ff-7e8526fde7ec">